client/utils/currency/index.js

```diff
@@ -3,9 +3,27 @@
  */
 import { __, sprintf } from '@wordpress/i18n';
 import Currency, { getCurrencyData } from '@woocommerce/currency';
-import { find, trimEnd, endsWith } from 'lodash';
-
-const currencyData = getCurrencyData();
+import { find, trimEnd, endsWith, values } from 'lodash';
+
+const currencyData = [
+	...values( getCurrencyData() ),
```

Comment:
Oh, right – `getDataForCountry` isn't ideal though, since we're dealing only with currencies directly, not starting with a country (like the onboarding profiler is).

I'll look into it a bit, but I don't think this PR makes things any worse in this regard, so we can open it as a separate issue if there isn't an easy way to address it – what do you think?

Reply:
> `getDataForCountry` isn't ideal though, since we're dealing only with currencies directly, not starting with a country (like the onboarding profiler is).

Hmm, yeah you're right. But I think that means we need to ask the wc-admin devs to keep `getCurrencyData()` in wc-admin right? Or at least some form of it? Maybe just a `getCountryForCurrencyCode()` or something along those lines?

It seems `getDataForCountry` is being _removed_ in v3.1.0 — not just deprecated. Or at least that's how I read the notice they added.

Reply:
Yeah, looks like you're right (assuming you mean `getCurrencyData`) – [this doc](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-deprecated/#api) specifies that it's the version when it'll be removed, probably in 4-6 months at the rate the releases seem to be going.

Just looked into it, and I think we could **load currency info [from WC core](https://github.com/woocommerce/woocommerce/blob/trunk/i18n/locale-info.php)**, as was done [here](https://github.com/woocommerce/woocommerce-admin/pull/5969/files#diff-512e7bd9a37734b0a1e95b579a48226e1628f8ebf6b3a5ed7a7c80c8d0c4e55fR698) (from the PR where `getCurrencyData` was deprecated) – I'll see about doing that in this PR, since it's a more complete way to close #1389 anyway.

Reply:
Tracking this in a separate issue now, since it's adjacent but independent: https://github.com/Automattic/woocommerce-payments/issues/2251